apps/mull-ui/src/app/app.tsx

```diff
@@ -121,8 +122,11 @@ export const App = () => {
             </PrivateRoute>
           </PrivateRoute>
           <PrivateRoute path={ROUTES.CAMERA} component={TrashRecognitionPage} />
-          <PrivateRoute path={ROUTES.PROFILE.DISPLAY}>
+          <PrivateRoute>
             <PrivateRoute path={ROUTES.PROFILE.EDIT} component={EditProfilePage} />
+            <div className="page-container">
+              <PrivateRoute path={ROUTES.PROFILE.DISPLAY} component={UserProfilePage} />
```

Comment:
```suggestion
              <PrivateRoute exact path={ROUTES.PROFILE.DISPLAY} component={UserProfilePage} />
```
This makes it so that if we go on `/profile/edit`, it doesn't show both the edit page and the profile page